files/__addonLocation__/package.json

```diff
@@ -22,6 +22,8 @@
     "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
     "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
     "lint:js": "eslint . --cache",
+    "lint:prettier": "prettier --check .",
```

Comment:
do we need this if we have the eslint prettier plugin installed? 🤔 https://github.com/embroider-build/addon-blueprint/blob/main/files/__addonLocation__/.eslintrc.cjs#L17 or maybe we should move it out of eslint too?

Reply:
eslint-plugin-prettier doesn't work with gjs -- however, that'll be fixed in https://github.com/ember-cli/eslint-plugin-ember/pull/1920, which I added to the PR description.

Reply:
we can unblock by splitting up the commands tho